build.sbt

```diff
@@ -33,30 +35,32 @@ lazy val redis =
   project
     .in(file("redis"))
     .enablePlugins(BuildInfoPlugin)
-    .settings(stdSettings("zio-redis"))
     .settings(buildInfoSettings("zio.redis"))
+    .settings(scala3Settings)
+    .settings(stdSettings("zio-redis"))
     .settings(
-      libraryDependencies ++= Seq(
-        "dev.zio"                %% "zio-streams"             % Zio,
+      libraryDependencies ++= List(
+        "dev.zio"                %% "zio-streams"             % "2.0.1",
```

Comment:
Using a variable instead of a literal might be a good idea.

Reply:
I agree. In fact, I plan to consolidate all of the dependencies that way, but I'll do that in a follow-up.